tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_CONUS_3km_GFDLgrid_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.yaml

```diff
@@ -7,11 +7,11 @@ metadata:
 user:
```

Comment:
Should the description be updated to note that the physics suite being used is now RRFS_v1beta rather than GFS_v16?